src/main/java/Calculator.java

```diff
@@ -0,0 +1,79 @@
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
+public class Calculator {
+    Scanner scanner = new Scanner(System.in);
+    private final String ANSI_RED = "\u001B[31m";
+    private final String ANSI_RESET = "\u001B[0m";
+    private double tempArg;
+
+    public void numbersValidator() {
+        double input;
+        boolean inputIsValid = false;
+
+        do {
+            try {
+                input = scanner.nextDouble();
+                tempArg = input;
+                inputIsValid = true;
+            } catch (InputMismatchException e) {
+                System.out.print(ANSI_RED + "Wrong input! Try Again: " + ANSI_RESET);
+                scanner.next();
+            }
+        } while (!inputIsValid);
+    }
+
+    public double doCalc(double firstArg, double secondArg, String operator) {
```

Comment:
in the future, it is better not to save on names, but to use the full name: doClalc -> calculate(...), doCalculation(...), ...